guides/source/active_storage_overview.md

```diff
@@ -721,10 +722,14 @@ Analyzing Files
 
 Active Storage analyzes files once they've been uploaded by queuing a job in Active Job. Analyzed files will store additional information in the metadata hash, including `analyzed: true`. You can check whether a blob has been analyzed by calling [`analyzed?`][] on it.
 
-Image analysis provides `width` and `height` attributes. Video analysis provides these, as well as `duration`, `angle`, and `display_aspect_ratio`.
+Image analysis provides `width` and `height` attributes. 
+Video analysis provides these, as well as `duration`, `angle`, `display_aspect_ratio` and presence/absence of `video` and `audio` channels.
```

Comment:
```suggestion
Image analysis provides `width` and `height` attributes. Video analysis provides these, as well as `duration`, `angle`, `display_aspect_ratio`, and `video` and `audio` booleans to indicate the presence of those channels.
```

Reply:
I had added a line break here because the "Audio analysis" was in a new line. Should I remove the new line from that one too?

Reply:
Yes, looks good! :+1:

Reply:
Oops, I was only inspecting the line break...  In the above suggestion, though, I had added an oxford comma and slightly rephrased the last item under video analysis.  If you think that sounds good, could you include those changes too?